src/pages/CreatePool/index.tsx

```diff
@@ -353,54 +366,51 @@ export default function CreatePool(): React.ReactElement {
                 <Typography my={2}>
                   {t("createPoolTypeDescription1")}
                 </Typography>
-                {chainId && ![ChainId.ARBITRUM].includes(chainId) && (
-                  <Typography mb={2}>
-                    {t("createPoolTypeDescription2")}
-                  </Typography>
-                )}
                 <Typography mb={2}>
-                  {t("createPoolTypeDescription3")}
+                  {t("createPoolTypeDescription2")}
                 </Typography>
-                <ToggleButtonGroup
-                  value={poolType}
-                  color="secondary"
-                  size="large"
-                  exclusive
-                  onChange={(event, value: PoolType) => {
-                    if (value) {
-                      setPoolType(value)
-                      if (value === PoolType.Base) {
-                        setTokenInputs([tokenInputs[0], ""])
+                <FormControl fullWidth>
+                  <InputLabel id="pool-type-label">Choose Pool Type</InputLabel>
+                  <Select
+                    label="Choose Pool Type"
+                    id="pool-select"
+                    onChange={(e) => {
+                      if (e.target.value === PoolType.Base) {
+                        setPoolType(e.target.value as PoolType)
+                        setTokenInputs(["", ""])
                       } else {
-                        setTokenInputs([tokenInputs[0]])
+                        const pool =
+                          expandedPools.data.byAddress[e.target.value as string]
+                        if (!pool) {
+                          console.error("Unable to locate pool")
+                          return
+                        }
+                        const tokensLength = pool.tokens.length
```

Comment:
nit: I think this is more of `tokensCount` rather than `tokensLength`. Up to you!